Makefile

```diff
@@ -47,6 +47,7 @@ install-golang-lint:
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(TOOLS_BIN_DIR) v1.50.1
 
 fmt: install-goimports addlicense
+	terraform fmt -recursive
```

Comment:
I don't want to have to run this command in terminal when I get terraform fmt error. I should be able to run it via make.